### PR TITLE
Add JOptionPanel to SingleValueSummaryTab

### PR DESCRIPTION
**Pull request by dvdhoz**
Add JOptionPanel for incorrect input.

### CODE REVIEW

File: ensemble-view/src/main/java/hec/ensembleview/tabs/SingleValueSummaryTab.java [hunk 1]
```diff
@@ -184,13 +184,16 @@ private void setActionListeners() {
...
-            tryShowingOutput(getValuesFromMetricCollectionTimeSeries(value));
+            try {
+                MetricCollectionTimeSeries value = statComputationHelper.computeTwoStepComputable(getFirstStat(), getFirstTextFieldValue(),
+                        getSecondStat(), getSecondTextFieldValue(),
+                        getSummaryType() == SingleValueSummaryType.COMPUTEACROSSENSEMBLES ||
+                                getSummaryType() == SingleValueSummaryType.COMPUTECUMULATIVE);
+                getValuesFromMetricCollectionTimeSeries(value);
+                tryShowingOutput(getValuesFromMetricCollectionTimeSeries(value));
+            } catch (Exception ex) {
+                JOptionPane.showMessageDialog(this, ex.getMessage(), "Input Error", JOptionPane.ERROR_MESSAGE);
```
Comment (joshuarwillis) [inline ensemble-view/src/main/java/hec/ensembleview/tabs/SingleValueSummaryTab.java:195]: The code changes here look good. Just two very minor things that I noticed when testing the UI. (1) When entering a percentile value greater than 1, the error message is missing ...less than "OR" equal to 1...  (2) When I hit compute and the percentiles box is empty, a more useful error message might be something along the lines of "Percentile(s) less than or equal to 1 must be entered in the text field"

**Review by joshuarwillis — CHANGES_REQUESTED** [review]

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Statistics.java [hunk 2]
```diff
@@ -3,27 +3,43 @@
...
+    Statistics(String statName, String displayString) {
         this.statName = statName;
+        this.displayString = displayString;
+    }
+
+    public String getDisplayString() {
+        return displayString;
+    }
+
+    public static Statistics fromDisplayString(String string) {
```
[the remark this answers is not in the document]
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Statistics.java:35]: I reset the class to back to its original state.

[the remark this answers is not in the document]
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Statistics.java:35]: I was originally trying to see if I could get the stat names to change using display name but didn't work out

File: ensemble-view/src/main/java/hec/ensembleview/tabs/SingleValueSummaryTab.java (hunk 1 above)
Comment (dvdhoz) [reply ensemble-view/src/main/java/hec/ensembleview/tabs/SingleValueSummaryTab.java:195]: Concur.  Updated the Exception messages to include "or" and added exception message to cumulative

**Review by joshuarwillis — APPROVED** [review]
@dvdhoz looks good. Marking as approved.

**Pull request merged**